packages/analytics-browser/src/plugins/file-download-tracking.ts

```diff
@@ -30,75 +30,81 @@ export const fileDownloadTracking = (): EnrichmentPlugin => {
   const name = '@amplitude/plugin-file-download-tracking-browser';
   const type = 'enrichment';
   const setup = async (config: BrowserConfig, amplitude: BrowserClient) => {
-    /* istanbul ignore if */
-    if (!amplitude) {
-      // TODO: Add required minimum version of @amplitude/analytics-browser
-      config.loggerProvider.warn(
-        'File download tracking requires a later version of @amplitude/analytics-browser. File download events are not tracked.',
-      );
-      return;
-    }
-
-    /* istanbul ignore if */
-    if (typeof document === 'undefined') {
-      return;
-    }
-
-    const addFileDownloadListener = (a: HTMLAnchorElement) => {
-      let url: URL;
-      try {
-        // eslint-disable-next-line no-restricted-globals
-        url = new URL(a.href, window.location.href);
-      } catch {
-        /* istanbul ignore next */
+    // The form interaction plugin observes changes in the dom. For this to work correctly, the observer can only be setup
+    // after the body is built. When Amplitud gets initialized in a script tag, the body tag is still unavailable. So register this
+    // only after the window is loaded
+    // eslint-disable-next-line no-restricted-globals
+    window.addEventListener('load', function () {
```

Comment:
this direct access to window might cause an issue with using this SDK in chrome extensions